### PR TITLE
Update raiwidgets change log

### PR DESCRIPTION
**Pull request by gregorybchris**
New release of responsibleai and raiwidgets packages

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -24,6 +24,13 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.5.0
+
+- Support for causal, counterfactual, and error analysis tools
```
Comment (romanlutz) [inline CHANGES.md:29]: According to the guide above this should be grouped by category (see v0.4.0 for an example)

Also, what was the package renamed to? And which package? We ship several out of this repo.

Comment (gregorybchris) [reply CHANGES.md]: Oh I see how that works, updated. It looks like people haven't been updating the v-next section as intended.. I'll go through and try to add as many details as possible from the commit history.

**Review by riedgar-ms — APPROVED** [review]

File: CHANGES.md (hunk 1 above)
Comment (romanlutz) [reply CHANGES.md]: Yes we've been very sloppy about this...

File: CHANGES.md [hunk 2]
```diff
@@ -24,6 +24,19 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.5.0
+
+- new features
+  - added support for causal, counterfactual, and error analysis tools in responsibleai
+  - added model analysis widget to raiwidgets
```
Comment (imatiach-msft) [inline CHANGES.md:31]: missing: add regression support to error analysis

File: CHANGES.md [hunk 3]
```diff
@@ -24,6 +24,19 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.5.0
+
+- new features
```
Comment (imatiach-msft) [inline CHANGES.md:29]: shouldn't we break this between raiwidgets and responsibleai packages as separate release notes?  Also, this isn't how we released responsibleai package last time, it's not part of the release pipeline currently.

**Comment by gregorybchris** [comment]
Pull request closed since the responsibleai package was released manually

File: CHANGES.md (hunk 2 above)
Comment (gregorybchris) [reply CHANGES.md:24]: Added

**Comment by gregorybchris** [comment]
Leaving this PR open until it's decided how to split changelogs between the two packages in this repo

**Comment by imatiach-msft** [comment]
closing as this package version has been released already

**Pull request closed without merging**